example/src/AppReachRouter.js

```diff
@@ -0,0 +1,50 @@
+import { ServerLocation, LocationProvider, navigate, useLocation } from '@reach/router'
```

Comment:
Ik twijfel of we `AppReachRouter` nog als voorbeeld toe moeten voegen. Als het goed is release ik vandaag een versie van `@kaliber/router`

Reply:
Ja goeie, het principe werkt, maar dit example kunnen we makkelijk later releasen.

Reply:
https://www.npmjs.com/package/@kaliber/routing (is versie 0.0.1, maar dan kan je al wel kijken of het een beetje wil)

Reply:
Example met routing ga ik zsm toevoegen. Met de huidige release kunnen we deze module meteen in Jumbo gebruiken.